tests/test_upstream_issue.py

```diff
@@ -15,9 +15,6 @@ class TestUpstreamIssue(unittest.TestCase):
     def setUp(self):
         self.mock_config = {
             'sync2jira': {
-                'default_github_project_fields': {
-                    'storypoints': ('Estimate', 'customfield_12310243')
-                },
                 'map': {
                     'github': {
                         'org/repo': {'sync': ['issue'], 'github_project_fields': {}},
```

Comment:
You've removed the `default_github_project_fields` definition (and concept), but what remains here is an empty `github_project_fields`.  Shouldn't we have a test case which tests some non-empty ones??